src/clj/game/core/diffs.clj

```diff
@@ -71,7 +72,10 @@
     [(not (is-public? card side))
      (prune-null-fields (private-card card))]
     [(:hosted card)
-     (update card :hosted (partial mapv #(card-summary % state side)))]
+     (-> card
```

Comment:
Instead of always updating `:hosted`, maybe something like:
```clojure
    (-> (if (:hosted card) (update card :hosted (partial mapv #(card-summary % state side))) card)
        (playable? state side)
        (card-abilities-playable? state side)
        (prune-null-fields))))
```